sketcherMinimizer.cpp

```diff
@@ -3608,4 +3610,99 @@ int sketcherMinimizer::morganScores(const vector<sketcherMinimizerAtom*>& atoms,
     return n;
 }
 
+// interactions with m_minimizer and m_fragmentBuilder
+std::vector<sketcherMinimizerBendInteraction*> sketcherMinimizer::getBendInteractions() const
+{
+    return m_minimizer.getBendInteractions();
```

Comment:
If you _can_, propagate `&` on up. For instance, if `m_minimizer.getBendInteractions();` is returning a const reference to internal data, this can return a const reference to that same data.

Reply:
That makes sense. Those functions don't return a const reference, however I'm pretty sure they could. 

One question -- what is the purpose of the const overloads (that you mentioned in [this](https://github.com/schrodinger/coordgenlibs/pull/84/#discussion_r597936030) comment)? Why not just have the const getters like we would do for this function?

Reply:
Maybe the const-only version is all we need. You're right - let's start there.

The non-const overloads are to allow edit access to these data structures. Maybe if, for instance, then want to adjust the coordinates of the atoms.